theia/settings.py

```diff
@@ -25,7 +25,7 @@
     SECRET_KEY = '6n$fr-9vudyln(o=6jzv*pr_3b9(kajv=#&wq6ep4vm_s1m@6p'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('NAME').lower() != "production"
```

Comment:
Where is the env var "NAME" defined? I don't see that var on either local or production instances. There's one called `ENV` on production that's set to `production` and is not set in development. What's this supposed to be?

Reply:
Good question! Let's set that to "development." I'll also push a commit to set the staging one, too.